### PR TITLE
lint: Update black to version 24

### PR DESCRIPTION
**Pull request by tomtseng**
CircleCI lint is breaking because `black` got updated from version 22 to 24 in our CircleCI Docker image. New versions of `black` add additional lint rules. We should either re-lint our files or pin `black`'s version.

This PR re-lints our files.

### CODE REVIEW

File: compose/transfer/twogtp-and-cpp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Builds an image that can run gogui's twogtp command for connecting two GTP
```
Comment (AdamGleave) [inline compose/transfer/twogtp-and-cpp/Dockerfile:1]: Did this file get added to this PR by mistake? I don't see what it has to do with `black`

**Review by AdamGleave — COMMENTED** [review]
LGTM apart from 1 spurious file; please request re-review once addressed (or feel free to merge if someone else LGTMs).

File: compose/transfer/twogtp-and-cpp/Dockerfile (hunk 1 above)
Comment (tomtseng) [reply compose/transfer/twogtp-and-cpp/Dockerfile]: yup, deleted

**Review by AdamGleave — APPROVED** [review]
LGTM

**Pull request merged**